frontend/pages/sitemap/[language_code_dot_xml].js

```diff
@@ -111,14 +111,54 @@ const getStaticPageEntries = (staticPages) => {
 };
 
 const renderEntry = (BASE_URL, url, priority, changefreq, lastmod) => {
+  const languageCode = getLanguageCode(url);
   return `<url>
       <loc>${BASE_URL}${url}</loc>
+      ${getXHTMLLinkTag(BASE_URL, url, languageCode)}
       <priority>${priority ? priority : 1}</priority>
       <changefreq>${changefreq ? changefreq : "daily"}</changefreq>
       ${lastmod ? `<lastmod>${lastmod}</lastmod>` : ``}
     </url>`;
 };
 
+const getXHTMLLinkTag = (BASE_URL, url, languageCode) => {
+  if (languageCode === "en") {
+    return `<xhtml:link
+    rel="alternate"
+    hreflang="${languageCode}"
+    href="${BASE_URL}${url}"/>
+    <xhtml:link
+    rel="alternate"
+    hreflang="de"
+    href="${BASE_URL}/de${url}"/>`;
+  }
+  const englishURL = BASE_URL + "/" + getEnglishURLFromGerman(url);
```

Comment:
Since this is only used within the next condition, WDYT about moving it inside that `if`? 
```suggestion

  const englishURL = BASE_URL + "/" + getEnglishURLFromGerman(url);
```

Reply:
Sounds good to me.